### ISSUE TITLE
Речевые модули - Отработка негатива

### ISSUE THREAD

**Issue by AuthFailed** [AuthFailed/chat-flomaster#11]

**Comment by VaderMann** [comment]
На данный момент общих ограничений по вашему адресу нет, предлагаю вам продолжить проведение первичной диагностики, она займет еще немного вашего времени

Согласны?

**Comment by VaderMann** [comment]
Я понимаю вас и полностью согласен, что данные сложности никому не выгодны

Именно поэтому я максимально заинтересован в долгосрочном устранении сложности, которая у вас наблюдается

Предлагаю вам продолжить проведение диагностики в рамках чата

Вас устроит данный вариант развития событий?

**Comment by VaderMann** [comment]
Понимаю вас и максимально заинтересован в устранении всех тех.неполадок, которые у вас сейчас наблюдаются, диагностика может помочь в устранении вашей сложность

Я могу вам предложить принять технического специалиста в рамках платного сервиса, в таком случае стоимость вызова составит от 600 рублей, стоимость самих работ инженер определит и сообщит на месте 

Для формирования заявки потребуется уточнить:
-Ваш точный адрес;
-Желаемое время визита техника;
-Ваш актуальный номер телефона.

Или мы может продолжить диагностику в рамках чата

Какой вариант будет для вас наиболее предпочтительным?

**Comment by VaderMann** [comment]
Согласен с вами, ситуация действительно неприятная, однако данный момент ближайший доступный интервал имеется только на предложенную дату, т.е возможности с нашей стороны вызвать техника на более раннее время нет, благодарю за понимание

**Comment by VaderMann** [comment]
Да, я вас понимаю, если в результате визит техника сложность будет выявлена с нашей стороны, то визит техника и все работы будут бесплатными

**Comment by VaderMann** [comment]
Да, я вас понимаю, если в результате визит техника сложность будет выявлена с нашей стороны, то визит техника и все работы будут бесплатными

**Comment by AuthFailed** [comment]
Перенесено в [планер](https://planer.chrsnv.ru/spaces/issues/5e583276999e4698a27d703f8b7bfea0)
